### PR TITLE
[MB] [#3] add DB backup file to repo

### PR DESCRIPTION
**Pull request by milcab**
I added the backup file along with the queries I used to create them. they might come in handy later

### CODE REVIEW

**Comment by tomselden** [comment]
looks good!

**Review by tomselden — APPROVED** [review]
good work!

File: DB/createTables.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS USERS (
```
Comment (Philkizzy) [inline DB/createTables.sql:1]: According to Postgres documentation, I think table name is lowercase

File: DB/createTables.sql [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	chatroomId int,
+	mediaType valid_media_type,
+	FOREIGN KEY(chatroomId) 
+	REFERENCES CHATROOM(id),
+	FOREIGN KEY(userId) 
+	REFERENCES USERS(id)
+);
+
+
+
```
Comment (Philkizzy) [inline DB/createTables.sql:54]: Great Job, this is well done !

**Pull request merged**